server/jetty/src/main/java/io/deephaven/server/jetty/JettyConfig.java

```diff
@@ -26,6 +27,7 @@ public abstract class JettyConfig implements ServerConfig {
     public static final int DEFAULT_PLAINTEXT_PORT = 10000;
     public static final String HTTP_WEBSOCKETS = "http.websockets";
     public static final String HTTP_HTTP1 = "http.http1";
+    public static final String HTTP_STREAM_TIMEOUT = "http2.stream.idleTimeout";
```

Comment:
I think we should call this `http2.stream.idleTimeoutMillis`, or leave as-is and expect parsing to be done via `java.time.Duration#parse`

Reply:
I think I like Duration strings rather than big ints, but I'm hesitant to change this since we are pretty uniformly millis for other configs at this time.